concent_api/concent_api/settings/base.py

```diff
@@ -227,8 +229,11 @@
 # contact concent for forced getting task results.
 FORCE_ACCEPTANCE_TIME = 4 * 3600  # 4 hours
 
-# A global constant defining the length of the time window within which FileTransferToken message is valid.
-MAXIMUM_DOWNLOAD_TIME = 1800  # 30 minutes
+# A global constant defining the assumed default resource download rate.
+MINIMUM_UPLOAD_RATE = constants.DEFAULT_UPLOAD_RATE
+
+# A global constant defining the download timeout margin independent from the size of the result.
+DOWNLOAD_LEADIN_TIME = int(constants.DOWNLOAD_LEADIN_TIME.total_seconds())
```

Comment:
WTF is 'leadin' mean?

Reply:
Good question, I was wondering too, I think someone from golem should know as it was taken from golem_messages.

Reply:
https://en.oxforddictionaries.com/definition/us/lead-in
> lead-in
>
> An introduction or preamble which allows one to move smoothly on to the next part of something.

Here it just means the initial, constant time added to the timeout. That's because there's always some costant overhead involved in starting the transfer no matter how little data you transfer.